modifications/DataClass.h

```diff
@@ -13,85 +13,143 @@
 #include <qregexp.h>
 #include "GlobalDictionary.h"
 #include "QDir"
+#include <QDateTime>
+#include <SignatureVerifier.h>
+#include <QMessageBox>
 
-///This is the core class that handles the communications with the management server, stores all the data related to vehicle.
```

Comment:
why is this comment removed? Something like this could be useful for understanding the implementation.

Reply:
Ahh..I was trying to write very specific comments and generate a Doxygen document and then thought it would be too time-consuming. Maybe we can add this kind of documentation in future